libraries/aws_iam_policies.rb

```diff
@@ -35,19 +42,20 @@ def fetch_data
     pagination_options = {}
     catch_aws_errors do
       loop do
-        response = @aws.iam_client.list_policies(pagination_options)
+        response = @aws.iam_client.list_policies(@parameters, pagination_options) unless @parameters.empty?
```

Comment:
Ideally, I would have liked to have passed in `@parameters` always. However, the resource would hang if `@parameters` was empty.

Reply:
You could have fetch_data take parameters as argument that defaults to an empty hash. The unfiltered request still takes place if parameters are empty so code below should be the same?

  def fetch_data(parameters = {})
...
response = @aws.iam_client.list_policies(parameters, pagination_options)